saadeghi/daisyui
# [Bug fix] Fix the issue "Can't Override Input Size Attributes"

Fix the bug https://github.com/saadeghi/daisyui/issues/2651

Steps to reproduce: 
1. Checkout the branch
2. Run `npm install && npm run build && npm run playground`
3. In `src/playground/components/Component.astro`, update content to
```html 
<div>
  <input class="input input-sm input-primary m-10"></input>
</div>
```
Then update content to
```html
<div>
  <input class="input input-sm input-primary pl-8 m-10"></input>
</div>
```

**Expected result**: the left padding should be updated to 8 px
**Actual result**: the left padding is unchanged.

Testing Steps: 
1. Run `npm install && npm run build && npm playground`
2. In `src/playground/components/Component.astro`, update content to
```html 
<div>
  <input class="input input-sm input-primary m-10"></input>
</div>
```
Then update content to
```html
<div>
  <input class="input input-sm input-primary pl-8 m-10"></input>
</div>
```
**Expected result**: the left padding should be updated to 8 px
**Actula result**: the left padding was updated to 8 px
